drape_frontend/frontend_renderer.cpp

```diff
@@ -603,6 +668,33 @@ void FrontendRenderer::RenderScene(ScreenBase const & modelView)
   if (m_guiRenderer != nullptr)
     m_guiRenderer->Render(make_ref(m_gpuProgramManager), modelView);
 
+  if (m_useFramebuffer)
+  {
+    m_framebuffer->Disable();
+    m_renderer3d->Render(modelView, m_framebuffer->GetTextureId(), make_ref(m_gpuProgramManager));
+
+    m_isBillboardRenderPass = true;
+
+    // Test code to check ortho overlays in 3d mode
```

Comment:
What is test code? Is it a temporary code which should be removed? Or it is a production code? What does it test and why? More info is needed.